File: laramie-administracion-webapi/src/domain/entities/cuenta-contable.ts
export default class CuentaContable {

    id: number;
	codigo: string;
	nombre: string;
	orden: number;
	agrupamiento: string;

	constructor(
        id: number = 0,
		codigo: string = "",
		nombre: string = "",
		orden: number = 0,
		agrupamiento: string = ""
	)
	{
        this.id = id;
		this.codigo = codigo;
		this.nombre = nombre;
		this.orden = orden;
		this.agrupamiento = agrupamiento;
	}

	setFromObject = (row) =>
	{
        this.id = row.id ?? 0;
		this.codigo = row.codigo ?? "";
		this.nombre = row.nombre ?? "";
		this.orden = row.orden ?? 0;
		this.agrupamiento = row.agrupamiento ?? "";
	}

}

File: modules/administracion/models/cuenta_contable.py
from sqlalchemy import Column, BigInteger, String, Integer

import sys, os
sys.path.insert(0, os.path.join(os.path.dirname(__file__), "..", ".."))
from shared.database import Base


class CuentaContable(Base):
    __tablename__ = "administracion_cuentas_contables"

    id = Column(BigInteger, primary_key=True, autoincrement=True)
    codigo = Column(String(50), nullable=False)
    nombre = Column(String(250), nullable=False)
    orden = Column(Integer, nullable=False, default=0)
    agrupamiento = Column(String(50), nullable=True)
